docs/src/__examples__/Dialog/DEFAULT.tsx

```diff
@@ -44,9 +44,26 @@ export default {
       </>
     );
   },
-  info: {
-    title: "Default dialog",
-    description:
-      "Dialogs present a simple choice for users. They're displayed after users take action.",
-  },
+  exampleKnobs: [
+    {
+      component: "Dialog",
+      knobs: [
+        {
+          name: "description",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "title",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "lockScrolling",
+          type: "boolean",
+          defaultValue: false,
+        },
+      ],
+    },
+  ],
```

Comment:
Dialog loses content in playground, maybe add some default title and description. And how about adding the flag for open and close?